tests/integration/agents/supervisor/test_planner.py

```diff
@@ -163,7 +163,7 @@ def test_invoke_planner(
     planner_correctness_metric,
     answer_relevancy_metric,
 ):
-    """Tests the invoke_planner method of SupervisorAgent."""
+    """Tests that the supervisor agent's invoke_planner method"""
```

Comment:
I think this description is accidentally incomplete?

Reply:
corrected.